src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * Allowed configuring PDO-based cache pools via a new `cache.adapter.pdo` abstract service
  * Deprecated auto-injection of the container in AbstractController instances, register them as service subscribers instead
  * Deprecated processing of services tagged `security.expression_language_provider` in favor of a new `AddExpressionLanguageProvidersPass` in SecurityBundle.
+ * Added auto-configured method call for `LoggerAwareinterface` instances.
```

Comment:
Enabled autoconfiguration for `LoggerAwareinterface`?

Reply:
Fixed!